logger/src/logger.cpp

```diff
@@ -56,4 +57,20 @@ void logMessage(const std::string& message,
     logMessage(Loglevel::INFO, message, location);
 }
 
+void fetchNlogs(size_t noofLogs)
+{
+    std::fstream logfile("/tmp/logdata.txt", std::ios::trunc | std::ios::out);
```

Comment:
Proper error handling is missing. Please fix that.

Reply:
I have not got any error while opening file. The file is getting created if the file is not present.

Reply:
Error handling is required for negative case. Please handle the case for any failure with file handling.